### ISSUE TITLE
about training data

### ISSUE THREAD

**Issue by boxuLibrary** [donydchen/mvsplat#30]
Hi, i have a question about training data. Have you trained only one checkpoints with all the training data? I mean i don't want to train different checkpoints for different dataset? And i wonder if it is possible and how should i organize the dataset? if i need to write different dataloader? Thanks.

**Comment by donydchen** [comment]
Hi @boxuLibrary, thanks for your interest in our work.

We trained one checkpoint on RE10K, _i.e._, [re10k.ckpt](https://drive.google.com/drive/folders/14_E_5R6ojOWnLSrSVLVEMHnTiKsfddjU), which was then used to test on RE10K, ACID and DTU. We also trained another checkpoint on ACID, _i.e._, [acid.ckpt](https://drive.google.com/drive/folders/14_E_5R6ojOWnLSrSVLVEMHnTiKsfddjU). Typically the model performs the best when trained/fine-tuned on the specific dataset. Still, our model generalized reasonably well that you can try to test on other datasets using the 're10k.ckpt'. 

You can reuse the re10k data loader at [dataset_re10k.py](https://github.com/donydchen/mvsplat/blob/main/src/dataset/dataset_re10k.py). Just ensure that you provide the normalized camera intrinsic and OpenCV-style camera extrinsic. For more details, kindly refer to https://github.com/donydchen/mvsplat/issues/28 and https://github.com/donydchen/mvsplat/issues/1.

**Comment by boxuLibrary** [comment]
Thank you so much. I have another question. When I use the multi-GPU to train the code, I find only one GPS is used. Should I modify the code to support the multi-GPU training?

**Comment by donydchen** [comment]
Hi @boxuLibrary, this project is set to automatically use multiple GPUs when they are available, no additional modification is required. 

If you run this project on a SLURM system, you might refer to [here](https://lightning.ai/docs/pytorch/stable/clouds/cluster_advanced.html) for more details on how to train a pytorch_lightning based project on multiple GPUs.
